PP_APP/server/frontend/templates/about.html

```diff
@@ -44,107 +44,125 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
+                            <h4 class="text-white py-2">Filip Stamenkovic</h4>
+                            <p class="mb-0 text-white">
+                                <a class="about-link px-2" target="_blank" href="https://github.com/fstamenkovic">
+                                  <i class="fab fa-github fa-2x"></i>
+                                </a>
+                                <a class="about-link px-2" target="_blank" href="https://www.linkedin.com/in/fstamenk/">
+                                  <i class="fab fa-linkedin fa-2x"></i>
+                                </a>
+                            </p>
+                            <hr class="d-none d-lg-block mb-0 ml-0" >
                         </div>
                     </div>
                 </div>
             </div>
-        </div>
 
-        <div class="row justify-content-center no-gutters">
-            <div class="col-lg-6">
-                <img class="img-fluid" src="{% static 'img/demo-image-02.jpg' %}" alt="">
-            </div>
-            <div class="col-lg-6 order-lg-first">
+            <div class="col-lg-6 order-lg-first px-3 py-3">
                 <div class="bg-black text-center h-100 project">
                     <div class="d-flex h-100">
                         <div class="project-text w-100 my-auto text-center text-lg-right">
                             <h4 class="text-white">Ryland Sepic</h4>
-                            <p class="mb-0 text-white">Another example of a project with its respective description.
-                                These sections
-                                work well responsively as well, try this theme on a small screen!</p>
+                            <p class="mb-0 text-white">
+                              <a class="about-link px-2" target="_blank" href="https://github.com/rmsepic">
+                                  <i class="fab fa-github fa-2x"></i>
+                              </a>
+
+                              <a class="about-link px-2" target="_blank" href="https://www.linkedin.com/in/ryland-sepic/">
+                                 <i class="fab fa-linkedin fa-2x"></i>
+                              </a>
+                            </p>
                             <hr class="d-none d-lg-block mb-0 mr-0">
                         </div>
                     </div>
                 </div>
             </div>
         </div>
 
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+        <h1 class="align-items-center text-center">ACKNOWLEDGEMENTS</h1>
+        <h2 class="align-items-center text-center">&nbsp;</h2>
+
+        <div class="featured-text text-center text-lg-left">
+            <p class="text-black-80">
+                Speical thanks to our client <a>Eric Law</a>
```

Comment:
This is a great way to recognize the people who supported us!

Reply:
Sweet. I removed the links I had for their names. After I thought about it, I figured we should ask their permission before doing something like adding their links.